tests/serializers/models.py

```diff
@@ -41,8 +41,32 @@ class ParentChild2Serializer(nested_serializers.NestedHyperlinkedModelSerializer
 
     class Meta:
         model = Child2
+        fields = ('name', 'url')
+
+
+class ParentChild1Serializer2(nested_serializers.NestedHyperlinkedModelSerializer):
+    class Meta:
+        model = Child1
         fields = ('url', 'name')
 
+    url = nested_serializers.NestedHyperlinkedIdentityField(
+        lookup_fields=('parent__pk', 'pk',),
+        view_name='parent3-child1-detail'
```

Comment:
Pretty sure I meant to use child1 here, and child2 a couple lines below, as I'm using the relations for child1 and child2 in separate viewsets.